gdk.pinned

```diff
@@ -1 +1 @@
-4e97647d387f5e1eb8f403431a70f891ff837e45
\ No newline at end of file
+1bf07e48
```

Comment:
We're not using the long hashes anymore?

Reply:
It doesn't really matter either way? Do you have a preference?

Reply:
No preference really